src/main/java/org/synyx/urlaubsverwaltung/person/PersonServiceImpl.java

```diff
@@ -104,7 +104,8 @@ public void delete(Person person, Person signedInUser) {
         workingTimeWriteService.deleteAllByPerson(person);
         personRepository.delete(person);
 
-        LOG.info("Deleted person with id {} deleted by signed in user with id {}", person.getId(), signedInUser.getId());
+        final String status = person.isActive() ? "active" : "inactive";
+        LOG.info("Deleted {} person with id {} ({}) deleted by signed in user with id {}", status, person.getId(), person.getUsername(), signedInUser.getId());
```

Comment:
"Deleted .... deleted by signed in user" ließt sich etwas komisch.

Vorschlag:
```suggestion
        LOG.info("person with id {} ({}) and status {} deleted by signed in user with id {}", person.getId(), person.getUsername(), status, signedInUser.getId());
```